src/lib/Containers/WorksForYou.tsx

```diff
@@ -42,8 +41,8 @@ export class WorksForYou extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props)
 
-    const notifications: object[] = this.props.viewer.me.followsAndSaves.notifications.edges.map(edge => edge.node)
-    if (this.props.viewer.selectedArtist) {
+    const notifications: object[] = this.props.query.me.followsAndSaves.notifications.edges.map(edge => edge.node)
```

Comment:
Yes. I think that in most/all cases this can be avoided, as I initially did for a few queries, but I backed out of that because I didn’t want to change too much runtime code at this point.